system_files/desktop/shared/usr/libexec/bazzite-flatpak-manager

```diff
@@ -22,24 +22,6 @@ if [[ -f $VER_FILE && $VER = $VER_RAN ]]; then
   fi
 fi
 
-# Opt out of and remove Fedora's flatpak repo
-if grep -qz 'fedora' <<< $(flatpak remotes); then
-  /usr/lib/fedora-third-party/fedora-third-party-opt-out
-  /usr/bin/fedora-third-party disable
-  flatpak remote-delete fedora --force
-fi
-
-# Ensure Flathub is enabled
-flatpak remote-add --if-not-exists --system flathub /usr/etc/flatpak/remotes.d/flathub.flatpakrepo
```

Comment:
I think enabling Flathub by default was a nice touch. Any reason not to do this anymore?

Reply:
AFAIK fedora 39+ ships with unfiltered flathub by default

Reply:
@dylanmtaylor 
Because new ISO action will enable FlatHub by default.

ISO is the main supported install method, while rebase is unsupported, so there is no reason for this to exist.

@FlyinPancake 

It ships unfiltered flathub, but does not enable it by default, unless you agree to the prompt in Gnome Software.